app/build.gradle

```diff
@@ -61,7 +61,10 @@ dependencies {
     implementation 'com.google.android.gms:play-services-base:18.0.1'
     implementation 'com.google.firebase:firebase-inappmessaging:20.1.2'
     implementation 'com.google.dagger:hilt-android:2.38.1'
+    implementation 'androidx.legacy:legacy-support-v4:1.0.0'
     kapt 'com.google.dagger:hilt-android-compiler:2.38.1'
+    //noinspection LifecycleAnnotationProcessorWithJava8
```

Comment:
Is this comment still useful ?

Reply:
oh maybe i taked that from a previous pull and don't update it yet